pkg/controllers/gpu/controller.go

```diff
@@ -407,5 +407,11 @@ func getPodArgs(gdp *devicepluginv1.GpuDevicePlugin) []string {
 		args = append(args, "-resource-manager")
 	}
 
+	if gdp.Spec.PreferredAllocationPolicy != "" {
```

Comment:
@mythi , I have no chance to test if the operator works, could you help to test? Thanks.

Reply:
I can help *you* to test it. If you have managed to test the plugin, you have a change to test it with the operator too.

1. deploy cert-manager
2. make the operator image and the gpu plugin image available to your registry/cluster 
3. [deploy](https://github.com/intel/intel-device-plugins-for-kubernetes/tree/main/cmd/operator#installation) operator
4. [deploy](https://github.com/intel/intel-device-plugins-for-kubernetes/tree/main/cmd/operator#usage) GPU plugin
5. change the CR using, e.g., `kubectl edit GpuDevicePlugin gpudeviceplugin-sample`

Reply:
Thanks for your guide, I have validated the operator.